draft-sharabayko-srt-over-quic.md

```diff
@@ -274,13 +275,23 @@ It is also possible for QUIC to not apply any congestion control, relying on SRT
 If the bitrate of the original stream already exceeds network throughput, SRT would still try to deliver it, maintaining congestion and
 eventually breaking the SRT connection.
 
+It is worth mentioning that, in live streaming scenario it may be beneficial to move congestion control mechanisms outside of the protocol towards the encoder (payload producer),
+implementing a network adaptive encoding based on the telemetry provided by SRT and QUIC protocols.
```

Comment:
```suggestion
It is worth mentioning that in a live streaming scenario it may be beneficial to move congestion control mechanisms outside of the protocol 
towards the encoder (payload producer), implementing a network adaptive encoding based on the telemetry provided 
by the SRT and QUIC protocols.
```